app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -668,9 +668,9 @@ $navbar-collapse-max-height: 480px;
 //
 //##
 //** Background color on `.list-group-item`
-$list-group-bg: $sidebars-background;
+$list-group-bg: $white;
 //** `.list-group-item` border color
-$list-group-border: $border-grey;
+$list-group-border: transparent;
```

Comment:
Yeah +1 on this, with the current look changes made in https://github.com/diaspora/diaspora/pull/6345 looks horrible.